pkg/config/planargument.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+import (
+	"github.com/lunarway/shuttle/pkg/git"
+	"os"
+	"path"
+	"strings"
+)
+
+func isPlanArgumentAFilePlan(planArgument string) bool {
+	return strings.Index(planArgument, "/") == 0 || strings.Index(planArgument, "./") == 0 || strings.Index(planArgument, "../") == 0
+}
+
+func isPlanArgumentAPlan(planArgument string) bool {
+	return planArgument != "" && (git.IsGitPlan(planArgument) || isPlanArgumentAFilePlan(planArgument))
+}
+
+func getPlanFromPlanArgument(planArgument string) string {
+	switch {
+	case isPlanArgumentAFilePlan(planArgument) && isFilePath(planArgument, true):
```

Comment:
Could this not be omitted as it results in the same result as the `default` case, or am I missing something?

Reply:
Maybe you are right, since `isFilePath(planArgument, false)` should not match this case.